File: 스마트폰_레포트/Report_04/RadioButtonTest/app/src/main/java/com/example/radiobuttontest/MainActivity.java
package com.example.radiobuttontest;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.RadioGroup;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void OnClick(View view) {
        EditText edit_num1 = (EditText) findViewById(R.id.num1);
        EditText edit_num2 = (EditText) findViewById(R.id.num2);

        String s_num1 = edit_num1.getText().toString();
        String s_num2 = edit_num2.getText().toString();

        if (s_num1.length() <= 0 || s_num2.length() <= 0)
            Toast.makeText(getApplicationContext(),
                    "빈칸이 존재합니다.", Toast.LENGTH_SHORT).show();
        else {
            Integer num1 = Integer.parseInt(s_num1);
            Integer num2 = Integer.parseInt(s_num2);

            RadioGroup radio_group = (RadioGroup) findViewById(R.id.radio_group);
            switch (radio_group.getCheckedRadioButtonId()) {
                case R.id.add:
                    Toast.makeText(getApplicationContext(),
                            String.valueOf(num1 + num2), Toast.LENGTH_SHORT).show();
                    break;
                case R.id.sub:
                    Toast.makeText(getApplicationContext(),
                            String.valueOf(num1 - num2), Toast.LENGTH_SHORT).show();
                    break;
                case R.id.mul:
                    Toast.makeText(getApplicationContext(),
                            String.valueOf(num1 * num2), Toast.LENGTH_SHORT).show();
                    break;
                case R.id.div:
                    Toast.makeText(getApplicationContext(),
                            String.valueOf(num1 / num2), Toast.LENGTH_SHORT).show();
                    break;
                default:
                    Toast.makeText(getApplicationContext(),
                            "연산을 선택하지 않았습니다.", Toast.LENGTH_SHORT).show();
                    break;
            }
        }
    }
}

File: 스마트폰_레포트/Report_10/1번/MainActivity.kt
package com.example.simplecalculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    lateinit var edit1: EditText;  lateinit var edit2: EditText
    lateinit var btnAdd: Button;   lateinit var btnSub: Button;
    lateinit var btnMul: Button;   lateinit var btnDiv: Button
    lateinit var btnRem: Button
    lateinit var textResult: TextView
    lateinit var num1: String;    lateinit var num2: String
    var result: Float? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        edit1 = findViewById<EditText>(R.id.Edit1)
        edit2 = findViewById<EditText>(R.id.Edit2)
        btnAdd = findViewById<Button>(R.id.BtnAdd)
        btnSub = findViewById<Button>(R.id.BtnSub)
        btnMul = findViewById<Button>(R.id.BtnMul)
        btnDiv = findViewById<Button>(R.id.BtnDiv)
        btnRem = findViewById<Button>(R.id.BtnRem)

        textResult = findViewById<TextView>(R.id.TextResult)

        btnAdd.setOnTouchListener { view, motionEvent ->
            num1 = edit1.text.toString()
            num2 = edit2.text.toString()
            if (num1.length == 0 || num2.length == 0){
                Toast.makeText(this, "값을 입력하세요!", Toast.LENGTH_SHORT).show()
            }
            else{
                result = num1.toFloat() + num2.toFloat()
                textResult.text = "계산 결과 : " + result.toString()
            }
            false
        }

        btnSub.setOnTouchListener { view, motionEvent ->
            num1 = edit1.text.toString()
            num2 = edit2.text.toString()
            if (num1.length == 0 || num2.length == 0){
                Toast.makeText(this, "값을 입력하세요!", Toast.LENGTH_SHORT).show()
            }
            else{
                result = num1.toFloat() - num2.toFloat()
                textResult.text = "계산 결과 : " + result.toString()
            }
            false
        }

        btnMul.setOnTouchListener { view, motionEvent ->
            num1 = edit1.text.toString()
            num2 = edit2.text.toString()
            if (num1.length == 0 || num2.length == 0){
                Toast.makeText(this, "값을 입력하세요!", Toast.LENGTH_SHORT).show()
            }
            else {
                result = num1.toFloat() * num2.toFloat()
                textResult.text = "계산 결과 : " + result.toString()
            }
            false
        }

        btnDiv.setOnTouchListener { view, motionEvent ->
            num1 = edit1.text.toString()
            num2 = edit2.text.toString()
            if (num1.length == 0 || num2.length == 0){
                Toast.makeText(this, "값을 입력하세요!", Toast.LENGTH_SHORT).show()
            }
            else {
                result = num1.toFloat() / num2.toFloat()
                if(num1.toFloat() != 0.0f && num2.toFloat() != 0.0f) {
                    textResult.text = "계산 결과 : " + result.toString()
                }
                else{
                    Toast.makeText(this@MainActivity, "오류 발생! 0을 나눌 수 없습니다!", Toast.LENGTH_SHORT).show()
                }
            }
            false
        }

        btnRem.setOnTouchListener { view, motionEvent ->
            num1 = edit1.text.toString()
            num2 = edit2.text.toString()
            if (num1.length == 0 || num2.length == 0){
                Toast.makeText(this, "값을 입력하세요!", Toast.LENGTH_SHORT).show()
            }
            else {
                result = num1.toFloat() % num2.toFloat()
                if(num1.toFloat() != 0.0f && num2.toFloat() != 0.0f) {
                    textResult.text = "계산 결과 : " + result.toString()
                }
                else{
                    Toast.makeText(this@MainActivity, "오류 발생! 0을 나눌 수 없습니다!", Toast.LENGTH_SHORT).show()
                }
            }
            false
        }
    }
}